apps/aecore/lib/aecore/chain/chainstate.ex

```diff
@@ -19,26 +20,30 @@ defmodule Aecore.Chain.Chainstate do
   @type accounts :: AccountStateTree.accounts_state()
   @type oracles :: Oracle.t()
   @type naming :: NamingStateTree.namings_state()
+  @type channels :: ChannelStateTree.channel_state()
   @type chain_state_types :: :accounts | :oracles | :naming | :none
```

Comment:
Should be added `:channels` and  `:none` can be removed because is already declared in `transaction.ex`